deploy/README.md

```diff
@@ -14,3 +14,69 @@ The just defines the infrastructure for the express app. This includes:
   amount of pods to manage the load
 - [Config](https://kubernetes.io/docs/concepts/configuration/configmap/) - configurable values for the app
 - [Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) - secret config values for the app
+
+# Alerts
+Various custom Cloud Platform alerts have been set up for the Intranet, Intranet Archive, and Justice UK.
+The following sections are short runbooks to explain what each alert means, and actions that could be taken toremedy the issue.
+
+
+# SlowResponse
+Projects: `justice-gov-uk`
```

Comment:
Since these all apply to this project, can I suggest remove all lines where `Projects: *`?